content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -258,14 +260,16 @@ const Stop: FC<{
               {!hasContext && (
                 <TombstoneTitle
                   level={tombstoneHeadingLevel}
-                  id={dasherize(`${title}`)}
+                  id={dasherize(title)}
```

Comment:
I'm 95% sure we can take this out now, `title` really should be a string.

Reply:
Presumably it could be a string with spaces in it, in which case we'd want to include dashes for a valid `id`?

Reply:
Ah, I meant the string interpolation rather than `dasherize`.